plugins/libplugin.c

```diff
@@ -501,6 +502,7 @@ handle_getmanifest(struct command *getmanifest_cmd,
 		json_out_end(params, '}');
 	}
 	json_out_end(params, ']');
+	json_out_addstr(params, "dynamic", dynamic ? "true" : "false");
```

Comment:
json_add_bool?

Reply:
I would have to include a lightningd/ header, is it ok or libplugin should not depend on lightningd/ files ?

Reply:
Anyway, the use of an enum instead of a bool as proposed below settled this